pynicotine/shares.py

```diff
@@ -477,14 +477,15 @@ def is_hidden(folder, filename=None, entry=None):
         """Stop sharing any dot/hidden folders/files."""
 
         # If the last folder in the path starts with a dot, or is a Synology extended
-        # attribute folder, we exclude it
+        # attribute, snapshot or recycle bin folder, we exclude it
```

Comment:
This comment should preferably be more generic, so you don't have to update it if the list of hidden folders is updated in the future.

Perhaps something like this:

> If the last folder in the path starts with a dot, or uses the same name as well-known hidden folders, we exclude it